frontend/apps/reader/modules/readertypeset.lua

```diff
@@ -56,63 +59,68 @@ function ReaderTypeset:onReadSettings(config)
     -- Block rendering mode: stay with legacy rendering for books
     -- previously opened so bookmarks and highlights stay valid.
     -- For new books, use 'web' mode below in BLOCK_RENDERING_FLAGS
-    local block_rendering_default_mode = 3
-    self.block_rendering_mode = config:readSetting("copt_block_rendering_mode")
-    if not self.block_rendering_mode then
-        if config:readSetting("last_xpointer") then
+    if config:has("copt_block_rendering_mode") then
+        self.block_rendering_mode = config:readSetting("copt_block_rendering_mode")
+    else
+        if config:has("last_xpointer") then
             -- We have a last_xpointer: this book was previously opened
             self.block_rendering_mode = 0
         else
             self.block_rendering_mode = G_reader_settings:readSetting("copt_block_rendering_mode")
-                                            or block_rendering_default_mode
+                                     or 3 -- default to 'web' mode
         end
         -- Let ConfigDialog know so it can update it on screen and have it saved on quit
         self.ui.document.configurable.block_rendering_mode = self.block_rendering_mode
     end
     self:setBlockRenderingMode(self.block_rendering_mode)
 
     -- set render DPI
-    self.render_dpi = config:readSetting("render_dpi") or
-        G_reader_settings:readSetting("copt_render_dpi") or 96
+    self.render_dpi = config:readSetting("render_dpi")
+                   or G_reader_settings:readSetting("copt_render_dpi")
+                   or 96
     self:setRenderDPI(self.render_dpi)
 
     -- uncomment if we want font size to follow DPI changes
     -- self.ui.document:setRenderScaleFontWithDPI(1)
 
     -- set page margins
-    local h_margins = config:readSetting("copt_h_page_margins") or
-        G_reader_settings:readSetting("copt_h_page_margins") or
-        DCREREADER_CONFIG_H_MARGIN_SIZES_MEDIUM
-    local t_margin = config:readSetting("copt_t_page_margin") or
-        G_reader_settings:readSetting("copt_t_page_margin") or
-        DCREREADER_CONFIG_T_MARGIN_SIZES_LARGE
-    local b_margin = config:readSetting("copt_b_page_margin") or
-        G_reader_settings:readSetting("copt_b_page_margin") or
-        DCREREADER_CONFIG_B_MARGIN_SIZES_LARGE
+    local h_margins = config:readSetting("copt_h_page_margins")
+                   or G_reader_settings:readSetting("copt_h_page_margins")
```

Comment:
What's the exact rule you've used btw? We should probably add it to the style guide. (I slightly prefer or at the beginning of the line, I care less about the specific indentation, and in any case there's a big effort here to standardize it.)

Reply:
> (I slightly prefer or at the beginning of the line)

Mhhh, I really prefer if there is some indentation. As for the level of indentation, I'd say let us have some freedom on that, it may depends on what we 'or':
- if it's too long, I might put it before the `= `  
- if it's short, I might indent it further than the `= '
- or I may just indent it so some part aligns with a part in each other, for aesthetics or fun :)

I'm quite fine with the solution above for this specific case (the or'ed things align).

Reply:
> Mhhh, I really prefer if there is some indentation.

No, that's not at all what I meant! ;-) It should definitely have one level of indentation (4 spaces), or more.

I meant as in end vs beginning
```lua
bla or
  alb
```
```lua
bla
  or alb
```

Reply:
The idea is to align the *variables* on the same column (especially for assignments).

I'm more ambivalent about tests ;).

e.g., [clang-format](https://releases.llvm.org/11.0.0/tools/clang/docs/ClangFormatStyleOptions.html) `AlignOperands`, either `Align` or `AlignAfter` w/ `BreakBeforeBinaryOperators` set to `NonAssignment`. (I tend to prefer Align for C/C++, because of parens and stuff).